docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md

```diff
@@ -42,38 +40,22 @@ Download the following configuration files:
 
 {% capture lessoncontent %} 
 
-## Create a PersistentVolume
-
-MySQL and Wordpress each use a PersistentVolume to store data. While Kubernetes supports many different [types of PersistentVolumes](/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes), this tutorial covers [hostPath](/docs/concepts/storage/volumes/#hostpath).
+## Create PersistentVolumeClaims and PersistentVolumes
 
-**Note:** If you have a Kubernetes cluster running on Google Kubernetes Engine, please follow [this guide](https://cloud.google.com/kubernetes-engine/docs/tutorials/persistent-disk).
-{: .note}
+MySQL and Wordpress each require a PersistentVolume to store data.  Their PersistentVolumeClaims will be created at the deployment step.
 
-### Setting up a hostPath Volume
+Many cluster environments have a default StorageClass installed.  When a StorageClass is not specified in the PersistentVolumeClaim, the cluster's default StorageClass is used instead.
```

Comment:
quick link or note on "how to not use default storage class"? It's not always clear that in order to stop dynamic provisioning sometimes you have to make the SC=""

Reply:
It is explained in detail in the PVC page.